dashboard/app/controllers/data_docs_controller.rb

```diff
@@ -57,8 +57,13 @@ def update
 
   # DELETE /data_docs/:key
   def destroy
-    @data_doc.remove_serialization
-    @data_doc.destroy
+    begin
+      @data_doc.remove_serialization
+      @data_doc.destroy
```

Comment:
please use `destroy!`. it will raise errors if it fails, which we would want to know about: https://tanzu.vmware.com/content/blog/were-finally-getting-destroy-in-rails-4

Reply:
will do! thank you